src/main/java/forestry/plugins/compat/PluginNatura.java

```diff
@@ -106,6 +119,34 @@ public void registerItems(RegistryEvent<Item> itemRegistryEvent) {
 				));
 				return;
 			}
+			
+			
+			
+			if (itemName.matches("^.*nether_glowshroom\\d?$")) {
+								
+				complexConsumeSubItems(item, "shrooms", subItem -> {
+					
+					shrooms.add(subItem);
+					
+					// find large shroom block that matches subItem
+					final Block largeShroomBlock = 
```

Comment:
How can I find corresponding huge mushroom block for subitem (without touching Natura API)?